jnigen/example/pdfbox_plugin/jnigen.yaml

```diff
@@ -33,7 +33,15 @@ c_root: 'src/'
 c_subdir: 'third_party/'
 
 ## Root for generated Dart bindings.
-dart_root: 'lib/third_party/'
+dart_root: 'lib/src/third_party/'
+
+## Root package configuration, this allows to write bindings to $dart_root/pdmodel.dart instead of
+## $dart_root/org/apache/pdfbox/pdmodel.dart assuming the org.apache.pdfbox is common to all subpackages
+## included in this config.
+##
+## Since every package maps to dart file, even if all your classes are in one package A.B.C, root package
+## will be A.B and not A.B.C
+root_package: org.apache.pdfbox
```

Comment:
Maybe this should be called something else.

The goal is to avoid unnecessarily deep nesting (which is _not_ avoided in Java itself). (I don't like the Java approach either, but is unambiguous. Maybe we should keep the nesting?)

This is slightly related to output folder in a sense `dart_root`. (Should we rename `dart_root` to something like `output` Since it is just a path, and we can reason about it on the file/folder level. Or even 

```yaml
output:
  dart: path/where/dart/files/are/generated.dart
  c: path/where/c/file/is/generated.c
```
)

```yaml
output:
  dart:
    path: 'path/where/dart/files/are/generated.dart'
    single_file: true
    un_nest: 'org.apache.pdfbox'
  c: 'path/where/c/file/is/generated.c'
```